packages/core-sdk/test/integration/ipAsset.test.ts

```diff
@@ -12,112 +12,103 @@ import {
 } from "./utils/util";
 import { MockERC20 } from "./utils/mockERC20";
 import {
-  derivativeWorkflowsAddress,
   evenSplitGroupPoolAddress,
   mockErc20Address,
+  piLicenseTemplateAddress,
   royaltyPolicyLapAddress,
+  derivativeWorkflowsAddress,
   royaltyTokenDistributionWorkflowsAddress,
 } from "../../src/abi/generated";
 
 chai.use(chaiAsPromised);
 const expect = chai.expect;
 
 const pool = evenSplitGroupPoolAddress[homer];
-describe("IP Asset Functions ", () => {
+
+describe("IP Asset Functions", () => {
   let client: StoryClient;
   let noCommercialLicenseTermsId: bigint;
   let parentIpId: Hex;
+
   before(async () => {
     client = getStoryClient();
     const res = await client.license.registerNonComSocialRemixingPIL({
-      txOptions: {
-        waitForTransaction: true,
-      },
+      txOptions: { waitForTransaction: true },
     });
     noCommercialLicenseTermsId = res.licenseTermsId!;
   });
 
-  describe("Create IP Asset", async () => {
+  describe("Basic IP Asset Operations", () => {
     let childIpId: Hex;
-    it("should not throw error when register a IP Asset", async () => {
+
+    it("should register an IP Asset", async () => {
       const tokenId = await getTokenId();
-      const waitForTransaction: boolean = true;
-      const response = await expect(
-        client.ipAsset.register({
-          nftContract: mockERC721,
-          tokenId: tokenId!,
-          txOptions: {
-            waitForTransaction: waitForTransaction,
-          },
-        }),
-      ).to.not.be.rejected;
-      if (waitForTransaction) {
-        expect(response.ipId).to.be.a("string").and.not.empty;
-        childIpId = response.ipId;
+      const response = await client.ipAsset.register({
+        nftContract: mockERC721,
+        tokenId: tokenId!,
+        txOptions: { waitForTransaction: true },
+      });
+
+      expect(response.ipId).to.be.a("string").and.not.empty;
+      if (!response.ipId) {
+        throw new Error("IP ID not returned from registration");
       }
+      childIpId = response.ipId;
     });
 
-    it("should not throw error when register derivative", async () => {
+    it("should register derivative", async () => {
```

Comment:
👍, we should try to keep these descriptions as concise as possible

Reply:
Yes, I have been dragging those for a while, let me fix them

Reply:
Done!